modules/ipi-modify-a-disconnected-registry-config-yaml.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+
+// * installing-mirroring-creating-registry.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-modify-a-disconnected-registry-config-yaml_{context}"]
+= Modify the `install-config.yaml` file to use the disconnected registry
+
+The `install-config.yaml` file must contain the disconnected registry node's certificate and registry information.
+
+.Procedure
+
+. Add the disconnected registry node's certificate to the `install-config.yaml` file. The certificate should follow the `"additionalTrustBundle: |"` line and be properly indented, usually by two spaces:
++
+[source,terminal]
+----
+$ echo "additionalTrustBundle: |" >> install-config.yaml
+$ sed -e 's/^/  /' /opt/registry/certs/domain.crt >> install-config.yaml
+----
+
+. Edit the `install-config.yaml` file to provide the mirror information for the registry.
+.. Update the `pullSecret` value to contain the authentication information from your registry:
++
+[source,terminal]
+----
+$ pullSecret: '{"auths":{"<mirror_host_name>:5000": {"auth": "<credentials>","email": "you@example.com"}}}'----
+----
++
+For `<mirror_host_name>`, specify the registry domain name that you specified in the certificate for your mirror registry, and for `<credentials>``, specify the base64-encoded user name and password for your mirror registry.
+
+.. Add the `additionalTrustBundle` parameter and value:
++
+[source,yaml]
+----
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+  -----END CERTIFICATE-----
+----
++
+The value must be the contents of the certificate file that you used for your mirror registry, which can be an existing, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
+
+.. Define the subnets for the VPC to install the cluster in:
```

Comment:
I think we don't need this step here, this has no relationship with a disconnected install.

Reply:
Removed the following: 

.. Define the subnets for the VPC to install the cluster in:
+
[source,yaml]
----
subnets:
- subnet-1
- subnet-2
- subnet-3
----